playbook/app/pb_kits/playbook/pb_selectable_card/docs/_selectable_card_error.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import { Body, SelectableCard, Title } from '../..'
+
+const SelectableCardError = (props) => {
+  const [football, setFootball] = useState(false)
+  const [basketball, setBasketball] = useState(false)
+  const [baseball, setBaseball] = useState(false)
+
+  return (
+    <div>
+      <Title
+          dark
+          size={3}
+          text="What sports do you like?"
+      />
+      <br />
+      <SelectableCard
+          {...props}
+          checked={football}
+          dark
```

Comment:
Don't need to add `dark` explicitly to these components.  They get pulled out by `{...props}`